lib/vsc/utils/fancylogger.py

```diff
@@ -101,6 +174,13 @@
 
 DEFAULT_UDP_PORT = 5005
 
+# poor man's enum
```

Comment:
> how is the code below an enum instead of a namedtuple?

http://stackoverflow.com/a/1695250/459543

the (semantic) difference being that a namedtuple has independent fields which can be separately assigned, whereas an enum is just a bunch of constants